torch/csrc/autograd/functions/init.cpp

```diff
@@ -110,10 +110,11 @@ bool THPAutograd_initFunctions(PyObject* _unused)
 
   generated::initialize_autogenerated_functions();
 
-  THPObjectPtr parent(PyImport_ImportModule("torch._C"));
-  if (!parent) return false;
-  PyModule_AddObject(parent.get(), "_functions", module.release());
```

Comment:
do you need to incref this?

Reply:
I'll add an explicit `Py_INCREF` to make it more clear, but the `module.release()` releases ownership and PyModule_AddObject steals it.